ca/proto/ca.proto

```diff
@@ -10,6 +10,7 @@ service CertificateAuthority {
   rpc IssueCertificate(IssueCertificateRequest) returns (core.Certificate) {}
   rpc IssuePrecertificate(IssueCertificateRequest) returns (IssuePrecertificateResponse) {}
   rpc IssueCertificateForPrecertificate(IssueCertificateForPrecertificateRequest) returns (core.Certificate) {}
+  rpc GenerateOCSP(GenerateOCSPRequest) returns (OCSPResponse) {}
```

Comment:
I'm not sure I really agree. This was my first approach to the problem and it ends up being not as nice a solution. You require managing multiple gRPC connections and clients to talk to the same service in order to gain a separation of powers that I don't think really makes sense anymore. If there were services that were still _only_ making issuance RPCs I think I'd agree but since we now only have issuance+ocsp and only ocsp I think it makes sense for the services to provide those two sets of RPCs instead of splitting them and making callers manage multiple clients.

Reply:
The point of the separation of permissions is not that RA should be prevented from updating OCSP, it's that the OCSP Updater should be prevented from issuing certificates. So I think we want to maintain that separation; given that we want to maintain it, I think it's cleaner to have each RPC defined in just one service, even if that means that RA needs two clients that both talk to the CA.

Reply:
That separation still exists though, the ocsp-updater uses the OCSPGenerator service, which only includes the OCSP RPC, and the RA uses the CertificateAuthority service, which includes all the RPCs.

Reply:
Yep, the separation still exists, but I was saying that I think it's untidy to have the same RPC defined on two different services. That said, given that we're still maintaining the separation we need, I don't feel too strongly about this and am willing to let it stand as written.